index.html

```diff
@@ -0,0 +1,125 @@
+    </ul>
+  </div>
+  <nav class="top-var1">
+    <a href="https://github.com/nimplay">
+      <img class="icon-top-bar" src="./images/Github.png" alt="github"/></a>
+    <a href="https://twitter.com/NimrodAcosta">
+      <img
+        class="icon-top-bar"
+        src="./images/Icon-Twitter.png"
+        alt="twitter"/></a>
+    <a class="top-bar1-p" href="">Login/Register</a>
+  </nav>
+  <nav class="top-var2">
+    <div id="logo-icon">
+      <a href="./index.html"
+        ><img id="main-logo" src="./images/nimplayLogo.jpg" alt="Nimplay"
+      /></a>
+    </div>
+    <div id="main-bar2">
+      <a href="./about.html" class="text-orange top-text-var">About</a>
+      <a href="./index.html" class="text-gray top-text-var">Home</a>      
+      <a id="button-main-bar" href="https://nimplay.itch.io/">Jam</a>
+    </div>
+  </nav>
+</header> 
+  <section class="home" id="hero">
+    <div class="home-titles">
+      <h2 id="hello" class="text-orange">"Hello world"</h2>
+      <h1 id="maintitle" class="text-orange">NIMPLAY jam 2023</h1>
+    </div>
+    <div class="home-p-box">
+      <p class="text-gray">Enjoy the challenge of programming a video game in 15 days, 
+        show your skills in game development in our jam</p>
+    </div>
+    <div class="home-button-text">
+      <h2 id="home-date" class="text-gray">2023.09.15(Friday) - 30(Saturday). </h2>
+      <h3 class="text-gray">@NIMPLAY Academy</h3>
+    </div>
+  </div>
+  </section>
+  <section class="program">  
+    <h2 id="program-title" class="text-color">Main Program</h2>  
+    <div id="red-line"></div>
+    <div id="program-card">
+    </div>
+    <button class="card-buttom" aria-label="button">NIMPLAY jam 2023</button>
+    <p id="program-text-footer">SEE THE WHOLE PROGRAM</p>
+  </section>
+  <section class="participants">
+    
+   <div id="speaker-continer">
+    <h2 id="program-title" class="text-gray">Featured Guests</h2>  
+    <div id="red-line2"></div>
+    <div id="static">
+    <div class="speaker-data">
+      <img class="speaker-image" src="./images/Nimrod.jpg" alt="">
+      <div class="speaker-data2">
+      <h2 class="speaker-title">Nimrod Acosta</h2>
+      <p class="speaker-paragraph">Nimplay's CEO and video game developer .</p>
+        <div class="speaker-line"></div>
+      <p class="speaker-paragraph2">Four apps in Google Playstore and 3 years of experience 
+        programming video games, also 10 years like Music Coposition.</p>
+      </div>
+    </div>
+    <div class="speaker-data">
+      <img class="speaker-image" src="./images/speaker_02.png" alt="">
+      <div class="speaker-data2">
+      <h2 class="speaker-title">Nahomi Noh</h2>
+      <p class="speaker-paragraph">Professor in the Video Game University .</p>
+        <div class="speaker-line"></div>
+      <p class="speaker-paragraph2">As the main venue for new media art production in 
+        Korea. Lorem ipsum dolor sit amet consectetur.</p>
+    </div>
```

Comment:
According to the project requirements, the content of this section should be dynamically generated in its entirety.

<img width="487" alt="Screen Shot 2022-06-23 at 9 15 38" src="https://user-images.githubusercontent.com/82426682/175320937-306b30bb-baff-47b3-8fdf-3982c1abac06.png">

Reply:
sorry but the template have two speaker static and the rest appear clicking on the buttom. but all the main program is generated dynamically.  I think I did even more than what was required in the assignment in this section.